src/pbft/pbft.h

```diff
@@ -116,11 +118,13 @@ namespace pbft
     Pbft(
       std::shared_ptr<ChannelProxy> channels_,
       NodeId id,
+      std::unique_ptr<raft::LedgerEnclave> ledger_,
```

Comment:
raft is probably not the best namespace for LedgerEnclave now.

Reply:
it is not, i will move this in a different PR, moving it does not seem like it should be connected to this change